openstack/keppel-trivy/values.yaml

```diff
@@ -0,0 +1,47 @@
+global:
+  tld: DEFINED_IN_VALUES_FILE
+  region: DEFINED_IN_VALUES_FILE
+  registry: DEFINED_IN_VALUES_FILE
+
+owner-info:
+  support-group: containers
+  service: keppel
+  maintainers:
+    - Stefan Majewsky
+    - Sandro Jäckel
+    - Falk Reimann
+    - Muhammad Talal Anwar
+  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/openstack/keppel-trivy
+
+trivy:
+  image_version: "0.40.0"
+
+redis:
+  persistence:
+    enabled: true
+    accessMode: ReadWriteMany
+    size: 20Gi
+
+  redisPassword: DEFINED_IN_VALUES_FILE
+
+  config:
+    maxmemory: 1000mb # chosen to match resource limit below
+    maxmemory-policy: allkeys-lru
+    maxmemory-samples: 5  
+
+  resources:
+    limits:
+      memory: 1Gi
+      cpu: 100m
+    requests:
+      memory: 100Mi
+      cpu: 10m
```

Comment:
We are expecting around 3-5 GiB initially and maybe more in the future, so we need to give a substantially higher limit here. Let's start with `5Gi` for both memory request and limit and then we'll see where it goes from there.

Reply:
Also, going by https://redis.io/docs/reference/eviction/, it appears that we should prefer `maxmemory-policy: allkeys-random`.

Reply:
like we discussed I went with volatile-ttls